test-cypress/integration/setup/connectors-lucene.spec.js

```diff
@@ -0,0 +1,153 @@
+describe('Setup / Connectors - Lucene', () => {
+
+    let repositoryId;
+    const luceneConnectorName = 'lucene-test-index';
+    const fieldName = 'grape';
+    const connectorPropertyChain = 'http://www.ontotext.com/example/wine#madeFromGrape';
+    const uriType = 'http://www.ontotext.com/example/wine#Wine';
+    const connectorCreateToastMessage = 'Created connector ' + luceneConnectorName;
+    const connectorDeleteToastMessage = 'Deleted connector ' + luceneConnectorName;
+
+    before(() => {
+        repositoryId = 'repo' + Date.now();
+        cy.createRepository({id: repositoryId});
```

Comment:
I you don't need to have a single repository for the whole spec, then better change the hooks to `beforeEach` and `afterEach` in order to spawn a new repository for each test and avoid possible issues with dependent tests.

Reply:
hmm as I see it, you actually have dependencies between the tests. A connector is created in `Create a lucene connector` and then it's used in the following tests. My suggestion is to join those in a single test which to become a whole scenario: create connector -> copy it -> delete it. Having dependencies in the tests order is a bad practice because it makes them hard for maintainance and reason against. One should really read all the tests in order to understand how they depend on each other and how to not break them if wants to extend the spec somehow.

Reply:
Fixed with commit fe274051f6f1c19f89ac97e134e6086d2d98512c

Reply:
great

Reply:
Fixed in commit 331d9c9122e4c702a054325ddc791ec429a005dd